[llvm-objcopy] Add change-section-lma *+/-offset (#95431)
llvm/llvm-project: llvm/docs/CommandGuide/llvm-objcopy.rst

 Allow :program:`llvm-objcopy` to remove sections even if it would leave invalid
 section references. Any invalid sh_link fields will be set to zero.

.. option:: --change-section-lma \*{+-}<val>

>  Shift LMA of non-zero-sized segments by ``<val>``

Reviewer: ```suggestion
 Shift LMA of non-zero-sized segments by ``<val>``.
```

>  Shift LMA of non-zero-sized sections in the program header table by ``<val>``

Reviewer: ```suggestion
 Shift LMA of non-zero-sized segments by ``<val>``
```
How about this? Sections aren't in the program header table, only program headers (aka segments, which are in turn implicitly in the program header table).

Author: Added this suggestion to **Fixup3**

 Mark all defined global symbols as weak in the output.

> .. option:: --change-section-lma \*{+-}<val>

Author: Fixed

>  Shift LMA of non-zero-sized sections in the program header by ``<val>``

Author: Fixed

>  Currently only supports changing LMA for all sections by the same offset.

Author: Done
